app/controllers/discourse_automation/admin_automations_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module DiscourseAutomation
-  class AdminDiscourseAutomationAutomationsController < ::ApplicationController
```

Comment:
Those controllers were previously "public"?

Reply:
Unfortunately yes.

Reply:
Looks like it had an AdminConstraint on the route, so I think it was safe

https://github.com/discourse/discourse-automation/blob/d17a5aa26f67d25042686fe7ed54bff6d9bb63a3/plugin.rb#L101-L103

Reply:
Oh right. I have seen that and then it totally slipped my mind. 🤦🏻‍♀️